app/_includes/head.html

```diff
@@ -26,7 +26,8 @@
   <meta name="google-site-verification" content="CrU3zp02dNKTe8NSAipL4NCPkrIjDXG8fViTZ-MIzP4"/>
 
   <!-- New Relic -->
-  <script src="/assets/newrelic.js"></script>
```

Comment:
In this `/dist` directory, newrelic.js is nowhere to be found since our gulp task only copies the concatenated app.js. Since NewRelic is meant to be loaded before all other dependencies, I found Jekyll `include` to be the preferred approach and keep it inline.